about.md

```diff
@@ -11,38 +11,38 @@ image:
   credit: system58.photos by david.alan.kidd
   creditlink: http://system58photos.com/
 ---
-# Civic Data Alliance  
-## All the Data for We the People  
-The __Civic Data Alliance__ (CDA) is Louisville's [__Code for America Brigade__](http://brigade.codeforamerica.org/brigade/Civic-Data-Alliance/) and we are focused on advocating for inclusion, diversity, equity, accessibility, openness and transparency in civic tech and open data, interested in liberating, improving, gathering, defining, and reporting on public data and the state of civic tech, while fostering an inclusive and diverse civic tech community.   
+# Civic Data Alliance
+## All the Data for We the People
+The __Civic Data Alliance__ (CDA) is Louisville's [__Code for America Brigade__](http://brigade.codeforamerica.org/brigade/Civic-Data-Alliance/) and we are focused on advocating for inclusion, diversity, equity, accessibility, openness, transparency and accountablitiy in civic tech and open data. Civic Data Alliance is interested in ethically liberating, improving, gathering, defining, and reporting on public data and the state of civic tech, while fostering an inclusive and diverse civic tech community.  We belive that our government should uphold and advocate for ethical principles regarding Open Data and be accountable for their Open Data practices, including the ethical collection, implementaion and sharing of The People's data.  
```

Comment:
Do you think this could be worded so it includes broader technology initiatives? Expound on the civic tech portion?